charts/nm-vllm-grafana-dashboards/README.md

```diff
@@ -0,0 +1,79 @@
+# NM vLLM Graphana Dashboards
```

Comment:
@DaltheCow I tried to make these docs better, will you give them a read if you have a chance?

Reply:
Omg they are so much better, I think you spelled everything out wonderfully